services/user.service.js

```diff
@@ -0,0 +1,16 @@
+import { User } from '../db/models/user.js';
+import { EntityService } from './entity.service.js';
+
+export class UserService extends EntityService {
+  constructor() {
+    super(User);
+  }
+
+  async getById(id) {
+    return User.query().findById(id);
+  }
+
+  async search(filter) {
```

Comment:
It would be better if you add separate search/find/getMany method to entity service.

Reply:
smth like:
```js
getMany(filter, relations?, pagination?){
    ...
}
```